src/config.ts

```diff
@@ -2840,4 +2846,41 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://www.mintscan.io/gravity-bridge/txs/{txHash}',
 	},
+	{
+		rpc: 'https://rpc.assetmantle.one/',
+		rest: 'https://rest.assetmantle.one/',
+		chainId: 'mantle-1',
+		chainName: 'AssetMantle',
+		stakeCurrency: {
+			coinDenom: 'MNTL',
+			coinMinimalDenom: 'umntl',
+			coinDecimals: 6,
+			coinGeckoId: 'asset-mantle',
```

Comment:
Can't find this ID in CoinGecko's coins list v3. Is it coming soon? or could use 'pool:umntl' instead (and define which pool)?

Reply:
ticket for listing ongoing,
submit with `assetmantle` in the form